src/pycram/description.py

```diff
@@ -303,7 +294,7 @@ def get_mesh_filename(self, geometry: MeshVisualShape) -> str:
         else:
             raise LinkGeometryHasNoMesh(self.name, type(geometry).__name__)
 
-    def set_pose(self, pose: Pose) -> None:
+    def set_object_pose_given_link_pose(self, pose: Pose) -> None:
```

Comment:
That name sounds confusing

Reply:
hmm the thing is the link pose shouldn't be settable, but you can move the object such that the link has this given pose, and Iw ould like users to know that what they are doing is actually moving the whole object when ding this

Reply:
so there is the usual set_pose of the link but I did there was issue a warning that link poses are not settable.